src/backend/mailer.js

```diff
@@ -1,33 +1,48 @@
+/* eslint-disable no-empty-label */
+'use strict';
+
+const config = require('../../config.json');
+
 /**
- * Created by championswimmer on 1/8/16.
- * Modified by uttpal.
+ * Set these before requiring aws-sdk for all ENV vars to be loaded properly
  */
-"use strict";
+process.env.AWS_ACCESS_KEY_ID = process.env.AWS_ACCESS_KEY_ID || config.AWS_ACCESS_KEY_ID;
+process.env.AWS_SECRET_ACCESS_KEY = process.env.AWS_SECRET_ACCESS_KEY || config.AWS_SECRET_ACCESS_KEY;
+process.env.AWS_BUCKET = process.env.AWS_BUCKET || config.AWS_BUCKET;
+process.env.CLOUD_STORAGE = process.env.CLOUD_STORAGE || config.CLOUD_STORAGE;
 
-const Promise = require('bluebird');
+const promise = require('bluebird');
 const helper = require('sendgrid').mail;
-const config = require('../../config.json');
-const sg = require('sendgrid').SendGrid(process.env.SENDGRID_API_KEY || config.SENDGRID_API_KEY);
+const sg = require('sendgrid')(process.env.SENDGRID_API_KEY || config.SENDGRID_API_KEY);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of process.env. (no-process-env)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)